src/rpc/misc.cpp

```diff
@@ -650,6 +651,533 @@ UniValue echo(const JSONRPCRequest& request)
     return request.params;
 }
 
+bool getAddressFromIndex(const int &type, const uint160 &hash, std::string &address)
+{
+    if (type == 2) {
+        address = CRavenAddress(CScriptID(hash)).ToString();
+    } else if (type == 1) {
+        address = CRavenAddress(CKeyID(hash)).ToString();
```

Comment:
CRavenAddress could be swapped with DecodeDestination (not sure if .ToString() is needed)